src/Protocol/Relay.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * @copyright Copyright (C) 2020, Friendica
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Protocol;
+
+use Friendica\Content\Text\BBCode;
+use Friendica\Core\Logger;
+use Friendica\DI;
+use Friendica\Model\Search;
+
+/**
+ * Base class for relay handling
+ */
+final class Relay
```

Comment:
You don't need the final keyword. As I learned, it's not that helpful and it blocks mocking.

Reply:
Okay. This was just copy and paste :-)